flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/plan/nodes/CommonPythonCalc.scala

```diff
@@ -49,11 +49,11 @@ trait CommonPythonCalc {
   private[flink] def createPythonScalarFunctionInfo(
       rexCall: RexCall,
       inputNodes: mutable.Map[RexNode, Integer]): PythonFunctionInfo = rexCall.getOperator match {
-    case sfc: ScalarSqlFunction if sfc.getScalarFunction.getLanguage == FunctionLanguage.PYTHON =>
+    case sfc: ScalarSqlFunction if isPythonFunction(sfc.getScalarFunction) =>
```

Comment:
We can remove the if statement here, as the rexCall is always a call contains python function.
Would be great if you can also add some comments here that saying that the rexCall is always a call contains python function.